server/views/pages/about-not-complete.njk

```diff
@@ -0,0 +1,107 @@
+            classes: "govuk-button--secondary",
+            type: 'button'
+        }
+    ]
+} %}
+
+{% block bodyStart %}
+    <div id="top"></div>
+{% endblock %}
+
+{% block content %}
+    {{ super() }}
+    {% if data.assessmentAreas.versionUpdatedAt %}
+        <p class="govuk-body govuk-!-margin-top-5">{{ locale.detail.lastUpdated }}</p>
+    {% endif %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            {% if errors.domain %}
+            {%  set errorList = [] %}
+            {%  for error in errors.domain %}
+                {%  set errorList = (errorList.push({ text: locale.errors[error]}), errorList)    %}
+            {%  endfor %}
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: errorList
+                }) }}
+            {% endif %}
+
+            {% if data.deliusData.sentences.length > 0 %}
+            <h2 class="govuk-heading-m">Sentence information</h2>
+            <table class="govuk-table govuk-!-margin-bottom-6 sentence-info">
+
+                <tr class="govuk-body-s">
+                    <td class="govuk-padding-right--25">{{ locale.detail.sentenceHeading }}</td>
+                    <td class="govuk-padding-right--25">{{ locale.detail.endDateHeading }}</td>
+                    <td class="govuk-padding-right--25">{{ locale.detail.unpaidWorkHeading }}</td>
+                    <td class="govuk-padding-right--25">{{ locale.detail.rarHeading }}</td>
+                </tr>
+
+                {% for sentence in data.deliusData.sentences %}
+                    <tr class="govuk-table__row govuk-body-l {{ 'single-sentence' if data.deliusData.sentences.length == 1 }}">
+                        <td class = "govuk-table__cell govuk-padding-bottom--25">{{ sentence.description }} <br/> {{ sentenceLength(sentence.startDate, sentence.endDate, locale.common.sentence) }}</td>
+                        <td class = "govuk-table__cell govuk-padding-bottom--25">{{ sentence.endDate | formatSimpleDate }}</td>
+                        <td class = "govuk-table__cell govuk-padding-bottom--25">
+                            {% if sentence.unpaidWorkHoursOrdered > 0 %}
+                                {{ sentence.unpaidWorkHoursOrdered }} hours
+                            {% else %}
+                                No
+                            {% endif %}
+                        </td>
+                        <td class = "govuk-table__cell govuk-padding-bottom--25">
+                            {% if sentence.rarDaysOrdered > 0 %}
+                                {{ sentence.rarDaysOrdered }} days
+                            {% else %}
+                                No
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor  %}
+            </table>
+            {% endif %}
+
+            <div class="govuk-warning-text">
```

Comment:
Is there a reason you're not using the macro from the design system?

Reply:
Updated now, just done that way as it was a nice easy copy and paste from the GDS for me to demo at the time. Good spot, needed changing. I had forgot